actor/src/main/scala/org/apache/pekko/serialization/Serialization.scala

```diff
@@ -484,8 +484,7 @@ class Serialization(val system: ExtendedActorSystem) extends Extension {
   // include "com.google.protobuf.GeneratedMessage" = proto in configured serialization-bindings.
   private def checkGoogleProtobuf(className: String): Boolean = checkClass("com.google.protobuf", className)
 
-  // pekko-protobuf is now not a dependency of remote so only load if user has explicitly added it
-  // remove in v1.1
+  // pekko-protobuf-v3
```

Comment:
The method itself should probably also go? The v3 package is called `org.apache.pekko.protobufv3`.

Reply:
Maybe user still have an explicitly dependence of it?

Reply:
This function is still used inside Serialization.scala.

If it is ok, I can create a separate issue to investigate what happens if we remove this function but I suspect it could cause trouble for the code that uses pekko-protobuf-v3 jar if we remove it.

Reply:
just 1 test in org.apache.pekko.remote.artery.HandshakeRetrySpec -- could be an intermittent test issue, I'll run it again

Reply:
that was an intermittent failure - I have included the change in this PR now - so the function is now removed